_posts/2021-03-03-emb-comp.md

```diff
@@ -8,8 +8,10 @@ side_image: "/assets/posts/emb-comp-study.png"
 side_image_alt: "A series of example embedding visualizations."
 ---
 
-We are building a tool for people who work with high-dimensional embedding spaces, a type of data common to many fields such as natural language processing, computer vision, and computational biology. In order to establish how embedding users understand and use their data, we are interviewing embedding model experts across these domains - for anyone who has worked extensively with embedding models in a recent project, we would love to <a href="{{ page.studyLink }}" target="_blank">hear about your experiences</a>!
+<em>This study is no longer recruiting. During September 2021, we are conducting a follow-up study that you can read more about <a href="/2021/09/12/emblaze.html">here</a>.</em>
```

Comment:
I think jekyll has some way to create links to posts that automatically adjust to whatever URL format we may use in the future. Try something like this `{% post_url 2020-02-08-logo %}`.